tree/ntuple/v7/inc/ROOT/RDaos.hxx

```diff
@@ -111,21 +112,22 @@ public:
       FetchUpdateArgs() = default;
       FetchUpdateArgs(const FetchUpdateArgs&) = delete;
       FetchUpdateArgs(FetchUpdateArgs&& fua);
-      FetchUpdateArgs(DistributionKey_t &d, AttributeKey_t &a, std::vector<d_iov_t> &v, bool is_async = false);
+      FetchUpdateArgs(const DistributionKey_t &d, std::span<AttributeKey_t const> as, std::span<d_iov_t const> vs,
```

Comment:
To have a consistent style, let's move the `const` qualifier before the type name.  I'd also remove it from the `std::span<d_iov_t>`, as I think that its `iov_len` member is assigned as part of a fetch operation.
```suggestion
      FetchUpdateArgs(const DistributionKey_t &d, std::span<const AttributeKey_t> as, std::span<d_iov_t> vs,
```

(also fix it below, i.e. lines 124-125)